tests/unit/core/test_store.js

```diff
@@ -56,34 +57,33 @@ describe('core store middleware', () => {
     expect(_createLogger.called).toEqual(false);
   });
 
-  it('uses a placeholder store enhancer when devtools is not installed', () => {
-    const _window = {}; // devToolsExtension() is undefined
-    const enhancer = middleware({
-      _config: configForDev(true),
-      _window,
-    });
+  it('uses a placeholder store enhancer when devtools is not available', () => {
+    const _window = {}; // __REDUX_DEVTOOLS_EXTENSION__() is undefined
+    const enhancer = middleware({ _config: configForDev(true), _window });
+
     expect(typeof enhancer).toBe('function');
+
     const createStore = () => {};
     expect(typeof enhancer(createStore)).toBe('function');
   });
 
-  it('adds the devtools store enhancer in development', () => {
+  it('adds the devtools store enhancer when config enables it', () => {
     const _window = {
-      devToolsExtension: sinon.spy((createStore) => createStore),
+      __REDUX_DEVTOOLS_EXTENSION__: sinon.spy((createStore) => createStore),
```

Comment:
I'm not sure why we need to pass `(createStore) => createStore` into `sinon.spy`. It doesn't seem as if it's actually doing anything.

Reply:
I think it is only for reproducing the behavior of the function. Do you want me to remove it?

Reply:
Well, I found it confusing, so if it doesn't actually do anything maybe it's better to remove it to avoid similar confusion in the future.

Reply:
👍